tests/robustness/validate/patch_history_test.go

```diff
@@ -92,7 +92,7 @@ func TestPatchHistory(t *testing.T) {
 		{
 			name: "failed put remains if there is a matching persisted request, uniqueness allows for revision and return time to be based on watch",
 			historyFunc: func(h *model.AppendableHistory) {
-				h.AppendPut("key", "value", 100, infinite, nil, errors.New("failed"))
+				h.AppendPut("key", "value", 100, 200, nil, errors.New("failed"))
```

Comment:
Only reason we only change this return time?

Reply:
With introduction of MaxInt64 we no longer need need to provide infinite time, it should be set during preparation. https://github.com/etcd-io/etcd/pull/19679

Reply:
Can you remove this change, I will merge https://github.com/etcd-io/etcd/pull/19679 first

Reply:
IIRC, without these 2 changes, the unit test will fail. But it's ok as I will rebase on your PR right after you merge!

Reply:
Done

Reply:
It's still here

Reply:
Is it?

Reply:
Maybe it's a delay in the GitHub UI...

<img width="1680" alt="Screenshot 2025-03-27 at 10 13 22 AM" src="https://github.com/user-attachments/assets/5ff10661-e230-4898-abf3-b1d6bd5f9602" />